spec/helpers/applications_helper_spec.rb

```diff
@@ -85,6 +85,12 @@
     end
   end
 
+  describe "decode" do
+    it "should decode the html entities such as &amp; to &" do
```

Comment:
>I can't think of the case we don't want to decode it or potential security issues...any insights?

Is there a case where it could potentially inject a `<script>` tag? e.g. by decoding an HTML entity for `<`? I'm honestly not sure if this is a problem here but this area of code, where you're manipulating outside external data and injecting it onto a page, is the place to be careful for this kind of security issue.